mono/utils/mono-threads.h

```diff
@@ -464,7 +464,7 @@ mono_thread_info_describe_interrupt_token (THREAD_INFO_TYPE *info, GString *text
 gboolean
 mono_thread_info_is_live (THREAD_INFO_TYPE *info);
 
-HANDLE
+MonoThreadInfo*
 mono_threads_create_thread (MonoThreadStart start, gpointer arg, MonoThreadParm *tp, MonoNativeThreadId *out_tid);
 
```

Comment:
I think the best way to go is to:
 - not merge this change
 - unify attach in threads.c between `mono_attach_thread_full` and `create_thread`
 - duplicate the handle in this unified threads.c attach mechanism (possible since we have a valid `MonoThreadInfo` which is not going to die in the meantime)

I will work on that ASAP